modules/EnsEMBL/Web/JSONServer/SpeciesSelector/Compara_Alignments.pm

```diff
@@ -57,18 +58,18 @@ sub json_fetch_species {
     $t->{children}   = [];
     my @children;
 
-    foreach (sort keys %{$row->{'species'}}) {
-      my $url_name  = $hub->species_defs->production_name_mapping($_);
-      my $prod_name = $hub->species_defs->get_config($url_name, 'SPECIES_PRODUCTION_NAME');
+    foreach $_ (sort keys %{$row->{'species'}}) {
+      my ($prod_name, $url_name); 
       my $ancestral = 0;
       if ($_ =~/ancestral_sequences/i) {
+        ## Not a real species, so it's legit to do ucfirst here!
         $url_name = ucfirst($_);
-        $prod_name = $_;
         $ancestral = 1;
       }
-
-      next unless $prod_name;
```

Comment:
are we missing this line?

Reply:
No, because I don't assign $prod_name until line 72. We always have a production name at this point because that's what the keys of the hash are that we're looping through.